scripts/docker/buildpack-deps/Dockerfile.ubuntu.clang.ossfuzz

```diff
@@ -91,35 +92,38 @@ RUN set -ex; \
 	rm -rf /usr/src/libprotobuf-mutator
 
 # EVMONE
+WORKDIR cd /usr/src
```

Comment:
Definitely a typo, also not sure why this worked.

Reply:
That's definitely what happens, since `WOKRDIR` is basically just a `mkdir` and `cd` - what I was confused about was what happens to the 'second' directory in this case, i.e. `/usr/src`. Couldn't find anything in the official reference, so I would assume that it just gets ignored.